### ISSUE TITLE
image normilization 

### ISSUE THREAD

**Issue by jremillard** [matterport/Mask_RCNN#95]
Sorry, if this is dumb question! 
The resnet tensor flow model that is in tensorflow, expects the input images to be processed by the per_image_standardization function, that subtracts the mean and divides by variance. Mass_RCNN, only subtracts a fixed offset from the image channels, which is from the Keras version of resnet. It seems both projects are using the caffe weights, shouldn't image normalization be the same, since it needs to do whatever was done in caffe?

**Comment by waleedka** [comment]
I think you're right. I checked the Keras imagenet weights and found two differences. Dividing by variance as you mentioned, and also the fact that it expects colors in BGR order rather than RGB. I don't know how big of an issue that is, though, because you typically use the imagenet weights as a starting point and train further. It will likely require longer fine-tuning, though. In general, I would recommend starting with the COCO weights instead.

A proper solution would be to replace the current imagenet weights with ones adopted for the current model, or to update image normalization to match imagenet and re-train our COCO weights for the new normalization.

**Comment by John1231983** [comment]
Hello, I am using your coco pretrained model v2 for finetune my dataset. Do I need to normalize my image to zero mean and unit variance before send to the network? On other hands, do you use any pre-processing to achieved your coco pretrained model? AFAIK, the pre-processing must be same the way that used to create the pre-trained model
